gradle.properties

```diff
@@ -14,8 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-version = 1.3.0-SNAPSHOT
-geodeVersion = 1.2.0
+version = 1.2.1
```

Comment:
Version changes shouldn't be part of this commit!

Reply:
Those were part of @metatype 's changes in af0c22ff30527c3d9c8ea98322d60129c6c0e749 to bump the version. I believe I got them when I rebased this feature branch against develop.